ops/platform-dev/base/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: content-library
+  annotations:
+    iam.gke.io/gcp-service-account: content-library-template@content-library-template.iam.gserviceaccount.com
```

Comment:
is this a real project / service account? or just a dummy value? I _think_ it's the latter but if so maybe we can rename it more obviously

Reply:
That is not!

This is actually intended to be replaced by the kustomize tool via the `overlays/dev(prod)` directory with the same matching filename.

Reply:
The way kustomize (the tool we're using to deploy these manifests) works is we essentially create a base template and then create 'overlays' that can contain just the data we want to overwrite like this service account email here. This way we can use a single template for multiple environments